### ISSUE TITLE
Exception  "Path must be a string" - Version 2.10.3 (Node 6)

### ISSUE THREAD

**Issue by codeuniquely** [tjunnone/npm-check-updates#329]
Just got a new Laptop (Mac Book Pro)

Doing a fresh fresh install of everything - Node, Npm, Git, Sublime Text, npm-check-updates, ...

Cloned my existing project
npm install - worked OK,

wanted to run ncu to see if it had installed any newer modules (compare to the project on my old MBP)

It threw this exception ....

```
Unhandled rejection TypeError: Path must be a string. Received null
    at assertPath (path.js:7:11)
    at Object.resolve (path.js:1146:7)
    at getPackageDataFromFile (/usr/local/lib/node_modules/npm-check-updates/lib/npm-check-updates.js:417:33)
    at programRunLocal (/usr/local/lib/node_modules/npm-check-updates/lib/npm-check-updates.js:407:19)
    at programRun (/usr/local/lib/node_modules/npm-check-updates/lib/npm-check-updates.js:356:50)
    at tryCatcher (/usr/local/lib/node_modules/npm-check-updates/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/usr/local/lib/node_modules/npm-check-updates/node_modules/bluebird/js/release/promise.js:510:31)
    at Promise._settlePromise (/usr/local/lib/node_modules/npm-check-updates/node_modules/bluebird/js/release/promise.js:567:18)
    at Promise._settlePromise0 (/usr/local/lib/node_modules/npm-check-updates/node_modules/bluebird/js/release/promise.js:612:10)
    at Promise._settlePromises (/usr/local/lib/node_modules/npm-check-updates/node_modules/bluebird/js/release/promise.js:691:18)
    at Promise._fulfill (/usr/local/lib/node_modules/npm-check-updates/node_modules/bluebird/js/release/promise.js:636:18)
    at Promise._resolveCallback (/usr/local/lib/node_modules/npm-check-updates/node_modules/bluebird/js/release/promise.js:431:57)
    at Promise._settlePromiseFromHandler (/usr/local/lib/node_modules/npm-check-updates/node_modules/bluebird/js/release/promise.js:522:17)
    at Promise._settlePromise (/usr/local/lib/node_modules/npm-check-updates/node_modules/bluebird/js/release/promise.js:567:18)
    at Promise._settlePromise0 (/usr/local/lib/node_modules/npm-check-updates/node_modules/bluebird/js/release/promise.js:612:10)
    at Promise._settlePromises (/usr/local/lib/node_modules/npm-check-updates/node_modules/bluebird/js/release/promise.js:691:18)

```

node -v
    V6.10.0
npm -v
    3.10.10

My old MBP (which runs ncu just fine) has: 
```
npm-check-updates .    V2.8.8
node                   V6.3.0
npm                    V3.10.3
```

As soon as I updated npm-check-updates to V2.10.3 
it also failed ....

When I then downgraded by new MBP installation to V2.8.8 
it started working.

**Comment by hueper** [comment]
Hi @codeuniquely,

same has happened to me.
Then I recognised I've tried to run _npm-check-updates_ from a directory without a package.json. When I run npm-check-updates from a directory with a package.json it works like expected :-)

**Comment by codeuniquely** [comment]
Yes it crashes when you don't have a package.json in the folder that you're running it from.

but V2.8.8 doesn't crash when it executes in the same folder ....

and instead returns this message.
```
No package.json

Please add a package.json to the current directory, specify the --packageFile or --packageData options, or pipe a package.json to stdin.
```

**Comment by raineorshine** [comment]
Thanks for reporting! This appears to be a regression from https://github.com/tjunnone/npm-check-updates/pull/325.

@eighty4 Could you look into this? Must be a hole in the test coverage.

**Comment by raineorshine** [comment]
Fixed in PR #332 and published in [v.2.10.4](https://github.com/tjunnone/npm-check-updates/releases/tag/v2.10.4).
